Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG STELLAR_CORE_VERSION=18.5.0-877.d387c6a71.focal
-ARG HORIZON_VERSION=2.15.1-247
+ARG HORIZON_VERSION=2.16.0-250
 ARG FRIENDBOT_VERSION=horizon-v2.12.1
```

Comment:
Can you also bump the FRIENDBOT_VERSION fields in both files to match?

Reply:
what version should friendbot be at? it's 2.12.1 in both at present, is that the horizon version number, should it. be `FRIENDBOT_VERSION=horizon-v2.16.0` ?

Reply:
It should be same version as horizon. It must have been an accident we updated one and not the other.